src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -165,10 +165,11 @@
     :subtitle        subtitle}])
 
 (defn- transaction-details
-  [{:keys [estimated-time-min max-fees token-display-name amount to-network route
+  [{:keys [estimated-time-min max-fees to-network route
            transaction-type]}]
   (let [route-loaded?             (and route (seq route))
-        loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])]
+        loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
+        amount                    (rf/sub [:wallet/send-total-amount-formatted])]
```

Comment:
showing same `total-amount` on the `transaction-confirmation` screen as on the `input-amount` one